### PR TITLE
[Merged by Bors] - chore(RingTheory): move `Ideal.exists_pow_le_of_le_radical_of_fG`

### PR DESCRIPTION
**Pull request by erdOne**
renamed to `Ideal.exists_pow_le_of_le_radical_of_fg_radical`, and added
`Ideal.exists_pow_le_of_le_radical_of_fg`

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/RingTheory/Ideal/Operations.lean [hunk 1]
```diff
@@ -603,6 +603,20 @@ theorem pow_right_mono {I J : Ideal R} (e : I ≤ J) (n : ℕ) : I ^ n ≤ J ^ n
   · rw [pow_succ, pow_succ]
     exact Ideal.mul_mono hn e
 
+lemma sup_pow_le_pow_sub_pow {R} [CommSemiring R] {I J : Ideal R} {n m : ℕ} :
```
Comment (chrisflav) [inline Mathlib/RingTheory/Ideal/Operations.lean:606]:
```suggestion
lemma sup_pow_add_le_pow_sub_pow {R} [CommSemiring R] {I J : Ideal R} {n m : ℕ} :
```
maybe?

**Review by chrisflav — APPROVED** [review]
Could you please note in the PR description that you add `exists_pow_le_of_le_radical_of_fg` (or simply rename the title). Otherwise, LGTM.
(Can I somehow delegate the `maintainer merge` to you?)

File: Mathlib/Algebra/Homology/LocalCohomology.lean [hunk 2]
```diff
@@ -206,26 +206,13 @@ def idealPowersToSelfLERadical (J : Ideal R) : ℕᵒᵖ ⥤ SelfLERadical J :=
 
 variable {I J K : Ideal R}
 
-/-- The lemma below essentially says that `idealPowersToSelfLERadical I` is initial in
-`selfLERadicalDiagram I`.
```
Comment (chrisflav) [inline Mathlib/Algebra/Homology/LocalCohomology.lean:210]: This part of the docstring was lost.

**Review by chrisflav — APPROVED** [review]
Otherwise, LGTM

File: Mathlib/Algebra/Homology/LocalCohomology.lean (hunk 2 above)
Comment (erdOne) [reply Mathlib/Algebra/Homology/LocalCohomology.lean:210]: I think this is only relevant in the context of that proof so I removed it.

Comment (chrisflav) [reply Mathlib/Algebra/Homology/LocalCohomology.lean:210]: Ah, makes sense.

**Comment by chrisflav** [comment]
Thanks!
maintainer-merge

**Review by jcommelin — COMMENTED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**